tests/foreman/ui/test_capsule.py

```diff
@@ -444,3 +446,95 @@ def test_positive_capsule_puppetca_autosign(self):
 
         @CaseLevel: System
         """
+
+    @stubbed()
+    @tier3
+    def test_enable_infoblox_dns_plugin(self):
+        """Check Infoblox DNS plugin can be enabled on capsule
```

Comment:
` tests/foreman/ui/test_capsule.py` contains all the tests related to UI  capsule  page. 

So either we can rename this test like:
"""Check infoblox DNS feature is enabled on capsule"""
and assert: DNS should be listed under capsule features

OR

Add new test as suggested above and move all installer related tests where scenario is capsule to `tests/foreman/installer/test_infoblox.py`